cdap-app-templates/cdap-etl/cdap-etl-batch/src/main/java/co/cask/cdap/etl/batch/ETLMapReduce.java

```diff
@@ -196,6 +201,10 @@ public void beforeSubmit(MapReduceContext context) throws Exception {
     job.setNumReduceTasks(0);
   }
 
+  private static LookupProvider getLookupProvider(DatasetContext context) {
+    return new DatasetContextLookupProvider(context);
```

Comment:
Why need a one line method to wrap around the creation of `DatasetContextLookupProvider`?